templates/default/opencast-setenv.erb

```diff
@@ -43,6 +46,8 @@ GC=""
 #GC="${GC} -XX:+PrintGCApplicationStoppedTime -XX:+PrintGCApplicationConcurrentTime -XX:+PrintHeapAtGC"
 #GC="${GC} -XX:+UseGCLogFileRotation -XX:NumberOfGCLogFiles=20 -XX:GCLogFileSize=32M -XX:+UseG1GC"
 
-export EXTRA_JAVA_OPTS="$GC -Dorg.eclipse.jetty.server.Request.maxFormContentSize=1500000 -Dfile.encoding=UTF-8"
+# #DCE OPC-773 Publish AWS metrics
+export EXTRA_JAVA_OPTS="${EXTRA_JAVA_OPTS} -Dcom.amazonaws.sdk.enableDefaultMetrics $GC -Dorg.eclipse.jetty.server.Request.maxFormContentSize=1500000 -Dfile.encoding=UTF-8"
```

Comment:
This will publish the metrics to CloudWatch. The side effect is that they appear in the Opencast log if you set the log level to DEBUG (currently in WARN only).